include/deal.II/fe/mapping.h

```diff
@@ -347,6 +347,16 @@ class Mapping : public Subscriptor
   get_vertices(
     const typename Triangulation<dim, spacedim>::cell_iterator &cell) const;
 
+  /**
+   * Return the mapped vertices of a face.
+   *
+   * Same as above but working on a given face of a cell.
```

Comment:
Can you please augment the documentation a bit, saying that `face_no` relates to the number of the face within the cell?